README.rst

```diff
@@ -13,18 +10,9 @@ With a simple and powerful API you can create many objects with a single line of
     :target: https://badge.fury.io/py/model_mommy
     :alt: Latest PyPI version
 
-Install
-=======
-
-.. code-block:: console
-
-    pip install model_mommy
-
-
-Usage and Info
-==============
+**IMPORTANT:** model_mommy is no longer maintained and was replaced by [model_bakery](https://pypi.org/project/model-bakery/). Please, consider [migrating your project](https://model-bakery.readthedocs.io/en/latest/migrating_from_mommy.html) to use the new lib.
```

Comment:
In restructured format (.rst) links does not work like this. :( Markdown FTW. :D

Reply:
My bad, I forgot the readme wasn't markdown. I'll fix the link here :+1: